printer_mods/sse018/Octopus_DIN_Rail_bracket/README.md

```diff
@@ -0,0 +1,17 @@
+# A DIN Rail bracket for BTT Octopus board - Voron 2.4
+
+This is a DIN Rail bracket for Bigtreetech Octopus board.
+
+Use default pcb_din_clip.stl x2, and print two parts and secure with m2 self-tapping screw.
+
+Octopus board can be secured to this bracket with M3 screw, without nuts.
+
+![Screenshots](./img/octobk-00.png)
+
+![Screenshots](./img/octobk-01.png)
+
+#### STL:
+![STL](./STL/Octopus_DIN_bracket_x2.stl)
```

Comment:
Please remove the exclamation mark from the beginning of these STL and CAD links, otherwise it makes it a broken image.  It should be just

    [STL](./STL/Octopus_DIN_bracket_x2.stl)

Reply:
Removed exclamation marks from STL and CAD links. Thank you.